tools/scripts/acr-build.ps1

```diff
@@ -183,8 +194,27 @@ $user = $credentials.username
 $password = $credentials.passwords[0].value
 Write-Debug "Using User name $($user) and passsword ****"
 
-$fullImageName = "$($script:Registry).azurecr.io/$($namespace)$($imageName):$($tagPrefix)$($sourceTag)$($tagPostfix)"
-Write-Host "Building image $($fullImageName)"
+# Get build root - this is the top most folder with .dockerignore
```

Comment:
main and master diverge here.
$prereleaseTag is a good indicator of what is being built in the latest commit. So I prefer to leave it here. We can discuss how to take this forward, if you prefer to remove it.

Reply:
Conclusion : Since main branch indicates the release candidate, pre-release tag for main should be RC.
In case the need arises where a feature is to be built but cannot be released as yet, and needs to be shared with partner, 'preview' branch should be spun off which cn be used for that purpose with a pre release tag like 'alpha' or 'beta' or 'preview'. 
Hence main should only contains commits which are approved to be released in the cadence like monthly or bi-monthly or LTS.
Nightly runs will determine the stability of main branch i.e. a release candidate.